diode-server/reconciler/ingestion_processor.go

```diff
@@ -346,6 +346,18 @@ func extractObjectType(in *diodepb.Entity) (string, error) {
 		return netbox.IpamIPAddressObjectType, nil
 	case *diodepb.Entity_Prefix:
 		return netbox.IpamPrefixObjectType, nil
+	case *diodepb.Entity_ClusterGroup:
+		return netbox.VirtualizationClusterGroupObjectType, nil
+	case *diodepb.Entity_ClusterType:
+		return netbox.VirtualizationClusterGroupObjectType, nil
```

Comment:
Should be `VirtualizationClusterTypeObjectType`

Reply:
```suggestion
		return netbox.VirtualizationClusterTypeObjectType, nil
```